.github/workflows/release.yml

```diff
@@ -0,0 +1,95 @@
+name: trino-python-client release
+
+on:
+  workflow_dispatch:
+
+env:
+  PYTHON_VERSION: "3.9.5"
```

Comment:
oddly specific version - any specific reason?

Reply:
Just wanted to hardcode some version to avoid breaking release workflow in the future.